README.md

```diff
@@ -21,4 +21,112 @@ An acme client that integrates with Kubernetes to automatically generate and ren
 
 ## Usage
 
-* _coming soon_
+### Initial setup
+
+At the moment kube-acme is to some extent limited by namespaces, and aspecially the inability
+of ingress to reference service from different namespace. This means that for an ingress
+to be able to properly work, a "local" (as in the "same namespace") kube-acme service is required.
+Still, to properly establish locks, a dedicated "acme" namespace is expected to exist.
```

Comment:
The "Still, to properly ..." line should probably be slightly more obvious. Perhaps add a line break just before it so it stands alone, and remove the "Still"?

`To properly establish locks, a dedicated "acme" namespace is expected to exist:` perhaps?